### ISSUE TITLE
Unnecessary logs??

### ISSUE THREAD

**Issue by maximjsx** [lulu5239/view-loggers#6]
Why is it getting logged if I view the page in the Browser?

![image](https://github.com/lulu5239/view-loggers/assets/114857048/0a5a2a6d-6f4b-4b8d-a4d8-622ba4213449)

**Comment by K0-RR** [comment]
Because that's how it should work? Discord is a web browser…

**Comment by maximjsx** [comment]
But why is the requested page favicon.ico then? Like this could be filtered... This is only happening if I open the actual site in my browser (not on discord.com)

**Comment by Ryfters** [comment]
i fixed this in my PR https://github.com/lulu5239/view-loggers/pull/5/

**Comment by lulu5239** [comment]
The favicon.ico page is automatically requested by browsers (when they search for an icon for the website). It lets you know when someone is opening the view-logger in a browser, it isn't that bad...

**Comment by Ryfters** [comment]
But it still triggers twice, and you just need to make it not trigger if the pathname is /favicon.ico

**Comment by maximjsx** [comment]
Yes also I dont want to know if the faveicon was requested... thats not the purpose of this. I just wanna know if a person requested a specific url not some random logs.
